go.mod

```
@@ -3,8 +3,8 @@ module go.aporeto.io/a3s
 go 1.20
 
 require (
-	go.aporeto.io/bahamut v1.144.0
-	go.aporeto.io/elemental v1.122.0
+	go.aporeto.io/bahamut v1.145.0
+	go.aporeto.io/elemental v1.122.1-0.20230127211758-51fb47178716
```

```suggestion
	go.aporeto.io/elemental v1.122.0
```
There's no real reason to leave the released versions train. When you remod stable libs, you can use `--version laster`   (diff https://github.com/PaloAltoNetworks/elemental/compare)/v1.122.0...master

I'll fix apomux's rmodlibs